src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import static core.basesyntax.ColorSupplier.RANDOM;
+
+public class FigureSupplier {
+
+    public Figure getRandomFigure() {
+        int ran = RANDOM.nextInt(5);
+        int a = RANDOM.nextInt(10) + 2;
+        int b = RANDOM.nextInt(10) + 2;
+        int c = RANDOM.nextInt(10) + 2;

```

Comment:
Don't create these values ahead - most them won't be always used. Do it on figures constructors calls. Also, put numeric values in constants (e.g. MAX_LENGTH)